src/app/container/container.component.html

```diff
@@ -80,17 +80,17 @@ <h3 class="pull-right">
     <h3>
       <div class="pull-right my-entry-button-group">
         <div class="btn-group" role="group">
-          <button id="publishToolButton" mat-raised-button color="secondary" (click)="publish()" [(ngModel)]="published"
-            [disabled]="publishDisable()" btnCheckbox [matTooltipShowDelay]="1000" [matTooltipPosition]="'before'" matTooltip="Publish/unpublish the tool">
+          <button id="publishToolButton" mat-flat-button color="accent" (click)="publish()" [(ngModel)]="published"
+            [disabled]="publishDisable()" btnCheckbox matTooltip="Publish/unpublish the tool" class="mr-2">
```

Comment:
Minor thought. Is it possible to detect the button state and either say `Publish the tool to make it visible to the public` or `Unpublish the tool to remove it from the public` or similar (removing the slash and providing more detail)?

Same with workflows

Reply:
It should be, I'll make the change.